src/cltk/sentence/__init__.py

```diff
@@ -0,0 +1 @@
+from .processes import *
```

Comment:
Star imports can have bad side effects downstream, and can create difficult bugs; please adjust to name the imports.

Reply:
Well, sadly, we have started doing this a lot in the library, so I guess it's okay to follow current precedence.

Reply:
I wasn't aware these happen elsewhere (will check). I think all star imports should be fixed.

Reply:
It is true that I took this kind of import in other similar `__init__.py`. It can be found in clkt.dependency, cltk.embeddings, cltk.lemmatize, cltk.sentence, cltk.stem, cltk.tokenizers.